nav.php

```diff
@@ -15,7 +15,8 @@
 	</a>
 	<div class="navbar-links">
 		<ul>
-			<li><a style="font-weight: 700;"><?php echo ($_SESSION['login_user'])?></a></li>
+			<li><a style="display: inline-block; font-weight: 700;"><?php echo ($_SESSION['login_user'])?></a></li>
+			<li2><a>|</a></li2>
```

Comment:
#added a seperator between links and the username output to give it more of a distinctive feel that it isn't a link